README.md

```diff
@@ -20,7 +20,7 @@ Package versions produced by official builds for this repo will automatically co
 The default build will automatically build all of the libraries that have been added to the repo. That said, given this repository is used to service packages, and you won't always want to service all of the packages at the same time, packages for libraries are not built by default. Whenever a package needs to be serviced, the following steps will need to be follwed:
 
 1. Determine the ServiceVersion to be used. When servicing a fix on a library package, the ServiceVersion (third part of the NuGet package version) will need to be bumped. This property is found in the library's src project. It's also possible that the property is not in that file, in which case you'll need to add it to the project and set it to 1. If the property is already in the source project, then just increment the servicing version by 1.
-2. Ensure that the source project has the property `<GeneratePackageOnBuild>true</GeneratePackageOnBuild>` set. This will ensure that the package for this library will be produced as part of the build.
+2. Ensure that the source project doesn't have the property `<IsPackable>false</IsPackable>` set. This will ensure that the package for this library will be produced as part of the build.
```

Comment:
Makes sense - so the assumption is that every src project in this repo is meant to be packable and we just target the `IsPackable` flag to enable and disable shipping.   We should test that this will do the right thing once we have multiple projects in the repo and one is referencing another that isn't shipping.  We'd expect that the one referenced would behave like it has a stable package version with it's last shipped version.